python/ray/dashboard/modules/metrics/dashboards/data_dashboard_panels.py

```diff
@@ -143,147 +143,145 @@
     # Ray Data Metrics (Inputs)
     Panel(
         id=17,
-        title="Input Blocks Received by Operator",
-        description="Number of input blocks received by operator.",
-        unit="blocks",
+        title="Input Blocks Received by Operator / Second",
+        description="Number of input blocks received by operator per second.",
+        unit="blocks/sec",
         targets=[
             Target(
-                expr="sum(ray_data_num_inputs_received{{{global_filters}}}) by (dataset, operator)",
-                legend="Blocks Received: {{dataset}}, {{operator}}",
+                expr="sum(rate(ray_data_num_inputs_received{{{global_filters}}}[30s])) by (dataset, operator)",
+                legend="Blocks Received / Second: {{dataset}}, {{operator}}",
             )
         ],
         fill=0,
         stack=False,
     ),
     Panel(
         id=18,
-        title="Input Blocks Received by Operator",
-        description="Byte size of input blocks received by operator.",
-        unit="bytes",
+        title="Input Blocks Received by Operator / Second",
+        description="Byte size of input blocks received by operator per second.",
+        unit="bytes/sec",
         targets=[
             Target(
-                expr="sum(ray_data_bytes_inputs_received{{{global_filters}}}) by (dataset, operator)",
-                legend="Bytes Received: {{dataset}}, {{operator}}",
+                expr="sum(rate(ray_data_bytes_inputs_received{{{global_filters}}}[30s])) by (dataset, operator)",
+                legend="Bytes Received / Second: {{dataset}}, {{operator}}",
             )
         ],
         fill=0,
         stack=False,
     ),
     Panel(
         id=19,
-        title="Input Blocks Processed by Tasks",
+        title="Input Blocks Processed by Tasks / Second",
         description=(
-            "Number of input blocks that operator's tasks " "have finished processing."
+            "Number of input blocks that operator's tasks have finished processing per second."
         ),
-        unit="blocks",
+        unit="blocks/sec",
         targets=[
             Target(
-                expr="sum(ray_data_num_task_inputs_processed{{{global_filters}}}) by (dataset, operator)",
-                legend="Blocks Processed: {{dataset}}, {{operator}}",
+                expr="sum(rate(ray_data_num_task_inputs_processed{{{global_filters}}}[30s])) by (dataset, operator)",
+                legend="Blocks Processed / Second: {{dataset}}, {{operator}}",
             )
         ],
         fill=0,
         stack=False,
     ),
     Panel(
         id=20,
-        title="Input Bytes Processed by Tasks",
+        title="Input Bytes Processed by Tasks / Second",
         description=(
-            "Byte size of input blocks that operator's tasks "
-            "have finished processing."
+            "Bytes of input blocks that operator's tasks have finished processing per second."
```

Comment:
do you think it's more clear to shorten it as you did? i thought "Bytes of blocks" didn't quite sound right to me for some reason.

```suggestion
            "Byte size of input blocks that operator's tasks have finished processing per second."
```